src/pages/[platform]/build-a-backend/graphqlapi/modify-amplify-generated-resources/index.mdx

```diff
@@ -1,7 +1,7 @@
 import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
 
 export const meta = {
-  title: 'Set up Amplify GraphQL API',
+  title: 'Modify Amplify-generated resources',
   description:
     "Learn more about how to get started with Amplify's API category",
   platforms: ['javascript', 'react-native', 'swift', 'android', 'flutter']
```

Comment:
@renebrandel This one looks like it was missed in the update as a couple things are diff from the new header

  platforms: [
                'android',
                'angular',
                'flutter',
                'javascript',
                'nextjs',
                'react',
                'react-native',
                'swift',
                'vue'
              ]
};

export const getStaticPaths = async () => {
  return getCustomStaticPath(meta.platforms);
};

export function getStaticProps(context) {
  return {
    props: {
      platform: context.params.platform,
      meta
    }
  };
}